crates/docmodel/src/document.rs

```diff
@@ -49,13 +49,6 @@ pub struct Document {
     /// this will be a subdirectory of `src_dir` named `build`.
     build_dir: PathBuf,
 
-    /// Arbitrary document metadata.
-    /// This has no effect on tectonic's build process.
-    /// Rather, allows users to add easily-accessible information to their documents.
-    ///
-    /// Tectonic's `show metadata` command displays this information.
-    pub metadata: Option<toml::Value>,
-
```

Comment:
Even if the current implementation doesn't do anything with the metadata, I think that we'd want to express it in the struct so that it can be roundtripped through TOML (de)serializations. And if someone writes a tool based on the `tectonic_docmodel` crate, we'd like to give them programmatic access to the information.

Reply:
Oops, fixed.